Web/Controllers/PaymentV2Controller.cs

```diff
@@ -152,23 +157,7 @@ public IActionResult PaymentAuthorizationCallback([FromQuery] string authorizati
     [HttpGet("accounts/{accountId}/payments/{paymentId}")]
     public async Task<IActionResult> PaymentDetails([FromRoute] string accountId, [FromRoute] string paymentId)
     {
-        if (_environment.IsProduction()) return NotFound();
-        try
-        {
-            var payment = await _aiiaService.GetOutboundPayment(User, accountId, paymentId);
-            return View("ObjectDetailsView", new ObjectDetailsViewModel("Outbound payment V2", payment, payment.Id));
-        }
-        catch (AiiaClientException)
-        {
-            try
-            {
-                var payment = await _aiiaService.GetInboundPayment(User, accountId, paymentId);
-                return View("ObjectDetailsView",  new ObjectDetailsViewModel("Inbound payment V2", payment, payment.Id));
-            }
-            catch (AiiaClientException)
-            {
-                return View("ObjectDetailsView");
-            }
-        }
+        var payment = await _aiiaService.GetOutboundPaymentV2(User, accountId, paymentId);
+        return View("ViewPaymentV2", new ViewPaymentV2ViewModel(payment));
```

Comment:
how do we know it's an outbound payment? do we have to do the same dance as before, i.e. try GetOutBoundPayment, if that fails, try GetInboundPayment?